package.json

```diff
@@ -68,13 +68,13 @@
     "typescript": "^4.1.2"
   },
   "dependencies": {
-    "@govtechsg/dnsprove": "^2.1.1",
-    "@govtechsg/document-store": "^2.0.0",
-    "@govtechsg/oa-did-sign": "^1.0.0",
-    "@govtechsg/oa-encryption": "^1.3.2",
-    "@govtechsg/oa-verify": "^5.1.0",
-    "@govtechsg/open-attestation": "^4.5.0",
-    "@govtechsg/token-registry": "^2.3.0",
+    "@govtechsg/dnsprove": "2.1.1",
+    "@govtechsg/document-store": "2.2.0",
+    "@govtechsg/oa-did-sign": "2.1.0",
+    "@govtechsg/oa-encryption": "1.3.2",
+    "@govtechsg/oa-verify": "7.1.0",
+    "@govtechsg/open-attestation": "4.5.0",
+    "@govtechsg/token-registry": "2.4.0",
```

Comment:
How come all the caret went missing? Dont have to fix the version, can use the `^` to allow minor/patch upgrades

Reply:
Done